### PR TITLE
[FIX] mail: fix margin issue when an activity is marked done

### PR DESCRIPTION
**Pull request by phenix-factory**
task-2280172

----

#

### CODE REVIEW

File: addons/mail/static/src/components/activity/activity.scss [hunk 1]
```diff
@@ -172,3 +172,12 @@ a:not([href]):not([tabindex]).o_Activity_detailsButton {
     color: gray('500');
 }
 
+.o_Activity_note, .o_mail_note_title + div {
+  p {
+    margin-bottom: 0;
+  }
+}
```
Comment (alexkuhn) [inline addons/mail/static/src/components/activity/activity.scss:179]: This stylerule is really not nice... kinda a way to say "but don't apply bootstrap margin on <p> in these (broadly) cases".

**Comment by seb-odoo** [comment]
This looks really bad on many levels, but after cleaning this up as much as possible, ok to put the crap must be kept in a legacy category.

**Comment by seb-odoo** [comment]
In this particular case, if this is "just" to reduce a margin somewhere, and only for legacy stuff, I would seriously challenge the spec to avoid doing it, but instead ensure future occurrences will look good by default.

**Comment by phenix-factory** [comment]
Ok, I'm confuse. What I should do here ?

File: addons/mail/static/src/components/activity/activity.scss [hunk 2]
```diff
@@ -172,3 +172,12 @@ a:not([href]):not([tabindex]).o_Activity_detailsButton {
...
+
+.o_mail_note_title {
+  margin-top: map-get($spacers, 2);
+}
```
Comment (alexkuhn) [inline addons/mail/static/src/components/activity/activity.scss:183]: Fix is nice. Would just re-organize a bit styles like this:

```scss
// ------
// Layout
// -------

// [AKU] put this style in alphabetic order
.o_Activity_note p {
   margin-bottom: 0;
}

// [AKU] put these styles at the end of category
// From python template
.o_mail_note_title {
   margin-top: map-get($spacers, 2);
}

.o_mail_note_title + div p {
   margin-bottom: 0;
}
```

_(without the [AKU] comments)_

Best to have simple and flat selectors, and the ones that apply on stuffs from python template is best to isolate from the rest (hence why I suggest putting it at the end).

I feel it's off to have this separation of component template and python templates... But this fix is acceptable considering it doesn't break old activity notes.

**Review by alexkuhn — CHANGES_REQUESTED** [review]

File: addons/mail/static/src/components/activity/activity.scss (hunk 2 above)
Comment (alexkuhn) [reply addons/mail/static/src/components/activity/activity.scss:183]: I'll make the change myself, just comment fyi ;)

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo retry

**Comment by alexkuhn** [comment]
@robodoo retry

**Pull request closed without merging**